tachyon/zk/plonk/examples/fibonacci/README.md

```diff
@@ -0,0 +1,9 @@
+# Fibonacci Circuits Test
+
+## Acknowledgements
+
+The three circuits and single chip originally come from the [halo2-examples](https://github.com/icemelon/halo2-examples/tree/6b8676487635169367e085a44e3b73c56290883f/src/fibonacci) repository by [icemelon](https://github.com/icemelon), demonstrating four different circuit designs for the Fibonacci sequence. `example4.rs` had not been implemented.
```

Comment:
Let's add a reason why `example4.rs` is not implemented! 
Maybe like ".. since it requires bigger size of evaluation domain for circuit tests..."